typed-racket-lib/typed-racket/types/subtype.rkt

```diff
@@ -705,6 +704,13 @@
                (and (not init-rest) (not init-rest*)
                     A)))]
      [_ (continue<: A t1 t2 obj)])]
+  [(case: Con (Con: t1-pre t1-post))
+   (match t2
+     [(Con: t2-pre t2-post)
+      (subtype-seq A
+                   (subtype* A t2-pre t1-pre obj)
+                   (subtype* A t1-post t2-post obj))]
+     [_ (continue<: A t1 t2 obj)])]
```

Comment:
The recursive calls to `subtype*` pass `obj`... but are we still talking about the same `obj`? If this is like recursively checking the domain and range of a function, those should not be `obj` but `#f` (or `-empty-obj`... I forget what the default "no object" value in this file is at the moment).

Reply:
actually you can just leave off `obj` it looks like (it's an optional arg)